### DISCUSSION TITLE
Bulk Importing multi-termination cables

### DISCUSSION THREAD

**Discussion by roys10** [netbox-community/netbox#14545] [Q&A] [answered]
How can I bulk import a cable that has a single side_a interface and multiple side_b destinations?
An example for what I mean https://www.youtube.com/watch?v=5JQlApaS8gM&t=375s, I want to create a connection like the one at 6:15 in the video just with bulk import.

**Comment by candlerb** [comment, marked as answer]
It's been asked before, e.g. #13206, #12387, #10319.

It's not possible to import multi-termination cables via CSV as the column headings are `side_(a|b)_(device|type|name)` only; and people have tried and failed to use JSON/YAML import. The capabilities of JSON/YAML import are not well documented and have very few test cases in the code.

However, you should be able to use the Django ORM (#13280, #12391) or the REST API (#13211, #11629, #11878 - note that the latter bug is about updating existing cables, not creating new ones)
